mxstbr/sharing
# Minor button text changes

- change 'Share on E-Mail' to 'Share by E-Mail'
- change medium size button text to icon + network
### Test plan
1. Choose "Medium" buttons, confirm that the text includes the icon and the network i.e. "<icon> Facebook"
2. Choose "Large" buttons, confirm that the text for E-Mail says "Share _by_ E-Mail" rather than "Share on E-Mail"
3. Paste the code for each case into html files to verify that the changes are reflected

Addresses https://github.com/mxstbr/sharingbuttons.io/issues/33
